superset-frontend/src/explore/components/ControlPanelsContainer.tsx

```diff
@@ -211,7 +212,36 @@ export class ControlPanelsContainer extends React.Component<ControlPanelsContain
 
     return (
       <Collapse.Panel
-        className="control-panel-section"
+        data-test="collapsible-control-panel"
+        css={theme => css`
+          margin-bottom: 0;
+          box-shadow: none;
+
+          &:last-child {
+            padding-bottom: ${theme.gridUnit * 10}px;
+          }
+
+          .panel-body {
+            margin-left: ${theme.gridUnit * 4}px;
+            padding-bottom: 0px;
+          }
+
+          .control-label {
+            margin-bottom: 0px;
+            position: relative;
+            span + span {
```

Comment:
I was wondering if we should add class names to these and target those?

Reply:
I went ahead and moved these styles to a better location. I'll sleep better now, thanks for the suggestion.

Reply:
I think they're a little more solid now too... see if you can break it 🤞